src/model/range.js

```diff
@@ -493,7 +493,10 @@ export default class Range {
 				// Without fix, the range would end up in the graveyard, together with removed element.
 				// <p>foo</p><p>[]bar</p> -> <p>foobar</p><p>[]</p> -> <p>foobar</p> -> <p>foo[]bar</p>
 				// <p>foo</p><p>bar[]</p>
```

Comment:
What does this comment mean? That last bit?

Reply:
It is the same as the above, just the input data is same. I was hoping that the explanation above will be enough to understand that these are similar cases with different input.

Reply:
Well maybe output is not exactly the same because range ends up in a different spot. I'll write whole example.